modules/@apostrophecms/schema/ui/apos/components/AposInputAttachment.vue

```diff
@@ -30,15 +30,15 @@ import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin.
 import AposSlatList from 'Modules/@apostrophecms/ui/components/AposSlatList';
 
 export default {
-  name: 'AposAttachment',
+  name: 'AposInputAttachment',
   components: {
     AposSlatList
   },
   mixins: [ AposInputMixin ],
   emits: [ 'upload-started', 'upload-complete' ],
   computed: {
     limitReached () {
-      return this.next._id;
+      return this.next.length >= this.field.max;
```

Comment:
We agreed that attachment fields would be singular, not arrays, at least until possibly beta.